docs: upload folder drag and drop (#3949)
vaadin/docs: articles/components/upload/index.adoc

endif::[]
--

> Upload also allows to drag & drop folders. This recursively uploads all files from the folder and its sub-folders. Note that the folder structure will be flattened and the names of uploaded files will not indicate in which folder they were in. Uploading folders is currently only supported when using drag & drop, but not when using the file picker.

Reviewer: ```suggestion
Upload can be configured to allow drag & drop uploading of folders. This uploads all files from the folder and its sub-folders recursively. Note that the folder structure will be flattened and the names of uploaded files will not indicate in which folder they were in. Uploading folders is currently only supported by drag & drop, not through the native file picker opened through the upload button.
```

Author: @rolfsmeds We don't have a config option for this, it just works out of the box.

Reviewer: Ah, indeed, lemme rephrase it.

```suggestion
Upload supports uploading folders via drag & drop. This uploads all files from the folder and its sub-folders recursively. Note that the folder structure will be flattened and the names of uploaded files will not indicate in which folder they were in. Uploading folders is currently not supported through the native file picker.
```

Author: Thanks!
